### DISCUSSION TITLE
🎥 Watch on-demand: Intro to Message Extensions and making your app intelligent with Microsoft Graph

### DISCUSSION THREAD

**Discussion by kamalshree** [microsoft/hack-together-teams#44] [Announcements]
On Wednesday, Jun 7, we discussed about Message Extensions and how to make your app intelligent with Microsoft Graph

![#4 - June 7 - Intro to Message Extensions and making your app intelligent with Microsoft Graph](https://github.com/microsoft/hack-together-teams/assets/3199282/40367918-076f-4765-9652-37f3b607a920)

In Part 1, @girliemac introduced the topic of Message Extension by showing off the preview of Message Extension Copilot, as well as providing some awesome and useful hack ideas using message extensions. After this, Sai Deepthi Kovvuru explained how Teams Apps are extensible across Microsoft 365.

As part of Part 2, @sebastienlevert and @maisarissi introduced Microsoft Graph with a demonstration

The session is available for on-demand viewing on the Microsoft 365 Developer YouTube channel at [https://www.youtube.com/watch?v=YPMPuxJllG4](https://www.youtube.com/watch?v=YTg63AnsOFo)

Check it out, and please don't hesitate to let us know if you've got any questions. Happy coding!

**Comment by waldekmastykarz** [comment]
If you didn't see the session live, head to YouTube and check it out now. It's packed with cool overview of one of the most engaging types of apps for Microsoft Teams and showing you how to tap into data and insights from Microsoft 365 using Microsoft Graph.
